frontend/src/__tests__/cypress/cypress/e2e/Application.cy.ts

```diff
@@ -0,0 +1,38 @@
+import { appChrome } from '~/__tests__/cypress/cypress/pages/appChrome';
+import {
+  asClusterAdminUser,
+  asDisallowedUser,
+  asProjectAdminUser,
+} from '~/__tests__/cypress/cypress/utils/users';
+
+describe('Application', () => {
+  it('should disallow access to the dashboard', () => {
+    asDisallowedUser();
+    appChrome.visit();
+    appChrome.shouldBeUnauthorized();
+    appChrome.findNavToggle().should('not.exist');
+  });
+
+  it('should not show Settings nav section for non-cluster admins', () => {
+    asProjectAdminUser();
+    appChrome.visit();
+    appChrome.findNavSection('Settings').should('not.exist');
+  });
+
+  it('should show Settings nav section for cluster admins', () => {
+    asClusterAdminUser();
```

Comment:
Can I suggest we either have a test for "cluster admin" or "product admin" -- Cluster Admin sounds like `cluster-admin` and that's technically different than just the admins of our product.

Reply:
ah yes. 
Having a distinct difference between product admin vs cluster admin is key. In fact almost all usage of cluster admin I did should be product admin.